config/default.yaml

```diff
@@ -6,6 +6,7 @@ proxy:
   print_service: https://return-case-doc.dev.ccd.reform.hmcts.net
   case_activity: http://localhost:3460
   payments: http://localhost:9999/payments
+  refdata: http://localhost:8090/refdata
```

Comment:
where is configuration for all the other environments?

Reply:
Can you point to the configuration files for the other environments?

Reply:
please have  a look at aat.tfvars, variables.tf, main.tf under folder infrstructure